tests-utils/src/main/scala/tech/beshu/ror/utils/containers/generic/ESWithRorPluginImage.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.utils.containers.generic
+
+import com.typesafe.scalalogging.StrictLogging
+import org.testcontainers.images.builder.ImageFromDockerfile
+import org.testcontainers.images.builder.dockerfile.DockerfileBuilder
+import tech.beshu.ror.utils.containers.ContainerUtils
+import tech.beshu.ror.utils.containers.DockerfileBuilderOps._
+import tech.beshu.ror.utils.containers.generic.EsWithRorPluginContainer.Config
+import tech.beshu.ror.utils.misc.Version
+
+object ESWithRorPluginImage extends StrictLogging {
+
+  private val rorConfigFileName = "readonlyrest.yml"
+  private val log4j2FileName = "log4j2.properties"
+  private val javaOptionsFileName = "jvm.options"
+  private val keystoreFileName = "keystore.jks"
+
+  def create(config: Config): ImageFromDockerfile = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=5012330)